app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const commentRoutes = require('./routes/comments');
+
+const app = express();
+
+mongoose.connect('mongodb://localhost:27017/mydb', { useNewUrlParser: true, useUnifiedTopology: true });
+
+app.use(bodyParser.json());
+
```

Comment:
I think you should also add here app.use(bodyParser.urlencoded({ extended: true }));
Also becuase we added in class